File: LV0/배열 만들기 3.js
function solution(arr, intervals) {
  let answer = [];
  for (let i = 0; i < intervals.length; i++) {
    let newArr = arr.slice(intervals[i][0], intervals[i][1] + 1);
    for (let j = 0; j < newArr.length; j++) {
      answer.push(newArr[j]);
    }
  }
  return answer;
}

File: LV0/배열 만들기 3.java
class Solution {

  public int[] solution(int n, int k) {
    int[] answer = {};
    for (int i = 1; i <= n; i++) {
      if (i % k == 0) {
        answer[answer.length] = i;
      }
    }
    return answer;
  }



